test/client/api.js

```diff
@@ -1021,6 +1021,18 @@ module.exports = config => {
     );
   };
 
+  ClientApi.prototype.grantTokensFromSessionToken = function (sessionTokenHex, oauthParams) {
```

Comment:
This is already in `master` under a slightly different name, `grantOAuthTokensFromSessionToken`.